src/Microsoft.AspNet.Testing/xunit/OperatingSystemVersions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Testing.xunit
+{
+    public static class OperatingSystemVersions
```

Comment:
I would name this class "WindowsVersions" because later we might have "LinuxVersions" and so on